### DISCUSSION TITLE
chocolateys only works on windows?

### DISCUSSION THREAD

**Discussion by blacktop** [goreleaser/goreleaser#3601] [Q&A]
I assume this can only work on Windows right?

**Comment by caarlos0** [comment]
AFAIK you can install `choco` on a linux machine and build there as well, but I haven't particularly tested it.

**Reply by caarlos0** [reply]
TODO to me: add it to goreleaser's test suite :)

**Reply by robinovitch61** [reply]
I stole a snippet to install choco on linux in my goreleaser-testing package: https://github.com/robinovitch61/goreleaser-testing/blob/603ecf99e1fc8832d51ffa3d7337f6d5af61471c/.github/workflows/release.yml#L30-L42

**Comment by blacktop** [comment]
I googled around for a little while and only found examples of how to install a windows choco client, though if it's just wrapping some simple API maybe we could add native support in goreleaser? but hopefully there's a linux client and if so should be able to at least compile a macOS client?

**Comment by blacktop** [comment]
https://github.com/chocolatey/choco/blob/develop/docker/Dockerfile.linux

**Comment by blacktop** [comment]
Can goreleaser run docker images as a pipeline?

**Reply by caarlos0** [reply]
it can't...

**Comment by blacktop** [comment]
I just tried running the `linux/amd64` image from dockerHUB and got a scarying looking crash/stack trace.  Not looking good 😞

**Reply by blacktop** [reply]
I suppose it might work on a linux host though, I haven't tried that yet.

**Comment by UlrichFrank** [comment]
Are there new experiences on this topic?
In the meanwhile there are builds and Docker Images available for chocolatey for linux and macOS 
Nevertheless I couldn't manage to get a choco package built with the expected content - it is always empty...

**Reply by caarlos0** [reply]
I think someone posted a working example on our discord... if you wanna join it: https://goreleaser.com/discord
